plugin/pkg/auth/authorizer/node/graph.go

```diff
@@ -132,21 +133,21 @@ var vertexTypes = map[vertexType]string{
 }
 
 // must be called under a write lock
```

Comment:
getOrCreateVertexLocked() should also  get specified under the comments before function name.